fast64_internal/mk64/__init__.py

```diff
@@ -0,0 +1,152 @@
+import bpy
+from bpy.types import PropertyGroup
+from ..f3d.f3d_material import createF3DMat
+from ..f3d.f3d_gbi import get_F3D_GBI
+from ..f3d.f3d_parser import getImportData, importMeshC
+from ..panels import MK64_Panel
+from ..utility import prop_split
+from bpy.utils import register_class, unregister_class
+from ..utility import raisePluginError
+from .f3d_course_parser import MK64F3DContext, parseCourseVtx
+
+
+class MK64_Properties(PropertyGroup):
+    """Global MK64 Scene Properties found under scene.fast64.mk64"""
+
+    # Import Course DL
+    EnableRenderModeDefault: bpy.props.BoolProperty(name="Enable Render Mode by Default", default=True)
```

Comment:
if I understand correctly you talk about bootupSceneOptions, DLExportSettings, DLImportSettings in OOT_Properties ? I don't see necessary the point because normally you wouldn't mix game importer/exporter

Reply:
Having the seperation makes intuitive sense I think, if youre trying to import this course you have these settings and then if youre trying to import x or y with the normal f3d parser you have its settings, it is what oot does anyways
But aside from that, if in the future mk64 gets more support its better to have a good foundation, with organization close to other game modes, I hope I'm making sense here